monkeytype/cli.py

```diff
@@ -168,6 +168,16 @@ def print_stub_handler(args: argparse.Namespace, stdout: IO, stderr: IO) -> None
     print(output, file=file)
 
 
+def print_list_handler(args: argparse.Namespace, stdout: IO, stderr: IO) -> None:
```

Comment:
Can you rename this to `list_modules_handler` so that it more closely matches the command that it implements?